src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,12 @@
 package mate.academy
 
 fun greetUsers() {
-    // implement function
+    while (true) {
+        print("Please enter a name (or press Enter to quit): ")
+        val name = readLine().orEmpty()
```

Comment:
FYI here could be minor refactoring using kotlin's built-in functionality
smth like this
        val name = readlnOrNull()
        if (name.isNullOrEmpty()) break
        
Your solution is also OK